prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -297,6 +297,18 @@ int efa_rdm_ep_post_user_recv_buf(struct efa_rdm_ep *ep, struct efa_rdm_ope *rxe
 
 	assert(rxe->iov_count > 0  && rxe->iov_count <= ep->base_ep.info->rx_attr->iov_limit);
 	assert(rxe->iov[0].iov_len >= ep->msg_prefix_size);
+
+	if (ep->user_rx_pkts_posted >= ep->efa_max_outstanding_rx_ops) {
```

Comment:
every user_rx_pkt will consume a user_rx pkt from the pool, right ? shouldn't it be protected by that pke alloc already?